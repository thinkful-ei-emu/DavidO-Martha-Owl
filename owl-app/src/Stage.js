import React from 'react';

export default function Stage (){
  const participants = [
    {
        id: 1,
        name: 'Koren Templeton',
        avatar:
            'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1',
        inSession: true,
        onStage: true
    },
    {
        id: 2,
        name: 'Caty Flucker',
        avatar:
            'https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1',
        inSession: true,
        onStage: false
    },
    {
        id: 3,
        name: 'Axe Kubicka',
        avatar: 'https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1',
        inSession: false,
        onStage: false
    },
    {
        id: 4,
        name: 'Frank Runciman',
        avatar:
            'https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1',
        inSession: true,
        onStage: false
    },
    {
        id: 5,
        name: 'Ashla Attwool',
        avatar:
            'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1',
        inSession: true,
        onStage: true
    }
];


const presenting = participants.filter(participant => participant.onStage === true );

const people = presenting.map((participant) => (
  <div>
    {participant.name}
   <img src = {participant.avatar} alt="peoples faces" />
</div>)
)

return (
 <div>
   <section>
     {people}
   </section>
 </div> 
)

}