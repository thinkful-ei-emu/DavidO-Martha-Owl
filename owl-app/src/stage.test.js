import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './Stage.js';


describe('<Stage />', () =>{
  it('renders without crashing', ()=> {
    const div = document.createElement('div');
    ReactDOM.render(<Stage />, div);
    ReactDOM.unmountComponentAtNode(div);

  });
});

it('renders the UI as expected', () => {
  const snapshot = renderer
    .create(<Stage />)
    .toJSON();
  expect(snapshot).toMatchSnapshot();  
});