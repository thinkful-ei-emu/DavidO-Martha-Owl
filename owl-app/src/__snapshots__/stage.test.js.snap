// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <section>
    <div>
      Koren Templeton
      <img
        alt="peoples faces"
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
    </div>
    <div>
      Ashla Attwool
      <img
        alt="peoples faces"
        src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      />
    </div>
  </section>
</div>
`;
